### PR TITLE
feat: post

### PR DESCRIPTION
**Pull request by techieshubham30**
- Add post
- Delete post
- Like post
- Dislike post
- Edit post

### CODE REVIEW

File: src/components/NewPost.js [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
+      comments: [],
+      content: "",
+    });
+  }
+  const dispatch = useDispatch();
   return (
     <div className="border-b border-gray-500 p-3 flex space-x-3 overflow-y-scroll scrollbar-hide">
       <img
-        src="https://images.pexels.com/photos/220453/pexels-photo-220453.jpeg?cs=srgb&dl=pexels-pixabay-220453.jpg&fm=jpg"
+        src={avatarURL}
```
Comment (mayuresh-tech) [inline src/components/NewPost.js:35]: Can you add a default URL if user doesn't have any avatar set

File: src/components/Post.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                </MenuButton>
+              }
+              transition
+            >
+              <MenuItem onClick={() => setModal(true)}>Edit</MenuItem>
+              <MenuItem onClick={() => deletePostHandler(_id, authToken)}>
+                Delete
+              </MenuItem>
+            </Menu>
+            {/* <HiOutlineDotsHorizontal className="h-5 text-[#6e767d] group-hover:text-[#1d9bf0]" /> */}
```
Comment (mayuresh-tech) [inline src/components/Post.js:111]: Commented code

**Review by mayuresh-tech — APPROVED** [review]
Good to merge!

File: src/components/NewPost.js (hunk 1 above)
Comment (techieshubham30) [reply src/components/NewPost.js:35]: Thanks for pointing it out, I will add it.

**Pull request merged**